.devcontainer/alma9/alma9-development.Containerfile

```diff
@@ -30,7 +31,7 @@ RUN dnf update -y && dnf install -y $APP_MAIN_DEPS && dnf clean all
 FROM install_main_deps as install_dev_deps
 
 COPY $APP_DEV_DEPS $APP_DEV_DEPS
-RUN $PIP install -r $APP_DEV_DEPS
+RUN $PIP install -r $APP_DEV_DEPS -f $APP_DOCS_DEPS
```

Comment:
@r0x0d not sure if this will work with the `../docs` part. But pip install can take multiple files